api-reference/40 Common Types/AIProvider/sendRequest.md

```diff
@@ -0,0 +1,149 @@
+---
+id: AIProvider.sendRequest
+type: function(params)
+---
+---
+##### shortDescription
+A function that sends a request to a language model.
+
+##### param(params): RequestParams
+Request parameters.
+
+##### return: Response
+An object with AI response.
+
+---
+---
+##### jQuery
+
+    <!-- tab: index.js -->
+    const aiIntegration = new DevExpress.aiIntegration({
```

Comment:
@marker-dao  is `aiIntegration` finally directly in DevExpress namespace? I though it's `DevExpress.common`